libs/locales/lib/en/translation.json

```diff
@@ -788,11 +788,12 @@
   "ai:Workers": "Workers",
   "ai:Workers: At least {{worker_cpu_cores}} CPU cores, {{worker_ram}} RAM, {{worker_disksize}} GB disk size for each worker ": "Workers: At least {{worker_cpu_cores}} CPU cores, {{worker_ram}} RAM, {{worker_disksize}} GB disk size for each worker ",
   "ai:x86_64": "x86_64",
-  "ai:You are approving multiple hosts. All hosts listed below will be approved to join the infrastructure environment if you continue. Make sure that you expect and recognize the hosts before approving.": "You are approving multiple hosts. All hosts listed below will be approved to join the infrastructure environment if you continue. Make sure that you expect and recognize the hosts before approving.",
-  "ai:You can either wait or investigate. A common issue can be misconfigured storage. Once you fix the issue, you can delete AgentServiceConfig to try again.": "You can either wait or investigate. A common issue can be misconfigured storage. Once you fix the issue, you can delete AgentServiceConfig to try again.",
+  "ai:You are approving multiple hosts. All hosts listed below will be approved to join the infrastructure environment if you continue. Make sure that you expect and recognize the hosts before approving.": "You are approving multiple hosts. All hosts listed will be approved to join the infrastructure environment, if you continue. Make sure that you expect and recognize the hosts before approving.",
+  "ai:You are deleting multiple resources. All resources listed below will be deleted if you continue. This action cannot be undone.": "You are deleting multiple resources. All resources listed will be deleted if you continue. This action cannot be undone.",
```

Comment:
not sure where this came from, but it needs to be removed

Reply:
Which line? Or both, 786+787?

Reply:
787

Reply:
```suggestion
```

Reply:
Addressed in 5c3f45f499e59b691a6ec8e6e56e2a81a7e1bde7

Reply:
Addressed in 5c3f45f499e59b691a6ec8e6e56e2a81a7e1bde7